assets/src/components/mapPage/routePropertiesCard.tsx

```diff
@@ -47,25 +46,25 @@ const DetailSection = ({
   children: JSX.Element
 }) => {
   return (
-    <details
-      className={`m-route-properties-card__details-section ${className}`}
-      open={isOpen}
-    >
-      <summary
-        className={`m-route-properties-card__summary${
-          isOpen ? " m-route-properties-card__summary--open" : ""
-        }`}
-        onClick={(event) => {
-          // preventDefault so that open/closed state is managed entirely by react
-          // see https://github.com/facebook/react/issues/15486
-          event.preventDefault()
-          toggleOpen()
-        }}
+    <div className="m-route-properties-card__details-section">
+      <button
+        className={`m-route-properties-card__toggle`}
+        aria-expanded={isOpen}
+        aria-controls={"details" + className}
+        onClick={toggleOpen}
+      >
+        <span id={"summary" + className}>
```

Comment:
**suggestion (non-blocking):** Using `useId` as part of the ID could help us avoid collisions in the future if we ever want to have multiple routes open at once, though I realize that's probably unlikely with our designs.